docs/src/pages/components/transitions/SimpleCollapse.js

```diff
@@ -25,50 +24,40 @@ const styles = theme => ({
     stroke: theme.palette.divider,
     strokeWidth: 1,
   },
-});
+}));
 
-class SimpleCollapse extends React.Component {
-  state = {
-    checked: false,
-  };
+function SimpleCollapse() {
```

Comment:
Just a preference can you make the function arrow functions?

Reply:
Looking at the other demos, this is they way it's done in those.

Reply:
Hmm, seems like there's a mix then. It should be fine to leave this as is then. I'm not sure which syntax we are preferring cc @oliviertassinari @eps1lon

Reply:
I really don't care. As long as both are named and no `this` is used inside the body they should be equivalent. Is there an observable difference between arrow functions and function declarations in this case?